javascript/javascript2/week1/spirit-animal/main.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+// Getting node for elements on the page
+
+//body
+const body = document.querySelector("body");
+
+//select input
+const dropdown = document.querySelector("select");
+
+//input
+const inputField = document.querySelector("input");
+
+//button
+const button = document.querySelector("button");
+
+//p for output
+const outputPar = document.createElement("p");
+body.appendChild(outputPar);
+
+//Create a function for random spirit animal generator
+function getSpiritAnimal() {
+  const nouns = [
+    "wolf",
+    "cat",
+    "eagle",
+    "pinguin",
+    "hamster",
+    "panda",
+    "lion",
+    "tiger",
+    "puma",
+    "zebra",
+  ];
+  const adjectives = [
+    "brave",
+    "lazy",
+    "fast",
+    "sleepy",
+    "tired",
+    "fluffy",
+    "cute",
+    "funny",
+    "chubby",
+    "crazy",
+  ];
+
+  let spiritAnimal = `The ${adjectives[Math.floor(Math.random() * 9)]} ${
+    nouns[Math.floor(Math.random() * 9)]
```

Comment:
So good! Super clever to randomize both the adjectives and nouns. 
 
One suggestion here is to replace `9` with e.g. `nouns.length` and `adjectives.length`, because let's say you decide to add more nouns and adjectives later; in this case you would have to also update the code down here every time. Just something to think about.

Reply:
Will fix it now :)